lib/stats.js

```diff
@@ -0,0 +1,37 @@
+/*!
+ * Copyright (c) 2023 Digital Bazaar, Inc. All rights reserved.
+ */
+import * as database from '@bedrock/mongodb';
+import dayjs from 'dayjs';
+
+/**
+ * Retrieves statistics from the database within a specified date range or
+ * for a specific date up to the end of the current date.
+ *
+ * @param {object} options - Options hashmap.
+ * @param {string} options.startDate - The start date of the range.
+ * @param {string} [options.endDate] - The end date of the range which is
+ *   optional. If an end date is not provided, the timestamp that corresponds
```

Comment:
I see that `day.js` can parse many different date formats.  https://day.js.org/docs/en/parse/parse

Since you are specifying the type as `string` here, I think it would be good to say that it's an ISO 8601 date string:

https://day.js.org/docs/en/parse/string

Reply:
Updated [4af23eb](https://github.com/digitalbazaar/bedrock-tokenization/pull/105/commits/4af23eb0ebb87ee1bd92daba444fe9bc0d546793). LMK if this looks okay.